docs/learn/learn-async-backing.md

```diff
@@ -7,103 +7,253 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
+  the number of parachain blocks a collator can produce that are not yet included in the relay
+  chain.
+
+- [`allowed_ancestry_len`](https://github.com/paritytech/polkadot-sdk/blob/f204e3264f945c33b4cea18a49f7232c180b07c5/polkadot/primitives/src/vstaging/mod.rs#L54):
+  the oldest relay chain parent a parachain block can be built on top of.
+
+Values of zero for both correspond to synchronous backing: `max_candidate_depth = 0` means there can
+be only one unincluded parablock at all times, and `allowed_ancestry_len = 0` means a parablock can
+be built only on the latest relay parent for that parachain. Initial values will be set to 3 (4
+unincluded parablocks at all times) and 2 (relay parent can be the third last).
+
+### Async Backing Diagram
+
+![async-backing](../assets/async-backing.png)
+
+The diagram assumes:
+
+- `max_candidate_depth = 2`, meaning that there can be a maximum of three unincluded parablocks at
+  all times
+- `allowed_ancestry_len = 1`, meaning parablocks can be anchored to the last or second-last relay
+  parent (i.e. collators can start preparing parablocks 6 seconds in advance)
+
+From left to right, parablock P1 is backed into the relay chain block R1 and included into R2. While
+P1 undergoes backing, collators can already start to generate P2, which will have R0 as a relay
+parent (showed with an `x`). Note how R0 can also be relay parent for P1 as long as in the
+unincluded segment there is a maximum of three unincluded parablocks. Parablock P2 can be backed in
+R2 (the same relay block where P1 is included) and included in R3. Collators can now use up to two
+seconds to generate parablocks. And so on, P3 can be generated while P2 is checked by backing
+groups, and P4 can be built while P3 undergoing backing. In 24 seconds, P1 to P3 are included into
+the relay chain.
+
+Note how there are always three unincluded parablocks at all times, i.e. compared to synchronous
+backing there can be multiple unincluded parablocks (i.e. [pipelining](#pipelining)). For example,
+when P1 undergoing inclusion, P2 and P3 are undergoing backing. Collators were able to generate
+multiple unincluded parablocks because on their end they have the
+[unincluded segment](#unincluded-segments), a local storage of not included parablock ancestors that
+they can use to fetch information to build new parablocks. On the relay chain side
+[perspective parachains](#prospective-parachains) repeats the work each unincluded segment does in
+tracking candidates (as validators cannot trust the record kept on parachains).
+
+Collating on top of unincluded parents means the only time limit to generate a parablock is how long
+it takes to back it (2 seconds). The 6-second relay chain block delay includes a backing execution
+timeout (2 seconds) and some time for network latency (time it takes to gossip messages across the
+entire network). Thus, even if a collator has >2 seconds to produce a block, the validators will
```

Comment:
im not sure what this sentence means, but I think you should use the more active voice.

"Collators are still limited to 2 seconds of block execution per parablock."